DEV-2/DEV-2Tests/CommonTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using DEV_2;
+using System;
+
+namespace DEV_2Tests
+{
+    [TestClass]
+    public class CommonTests
```

Comment:
only one method and construct was tested. It was needed to refactor and test others. Please read about Unit Testing